src/test/java/org/apache/commons/text/similarity/LongestCommonSubsequenceTest.java

```diff
@@ -89,6 +89,8 @@ void testLogestCommonSubsequence() {
         assertEquals("", subject.logestCommonSubsequence("", ""));
         assertEquals("", subject.logestCommonSubsequence("left", ""));
         assertEquals("", subject.logestCommonSubsequence("", "right"));
+        assertEquals("", subject.logestCommonSubsequence("l", "a"));
```

Comment:
This doesn't improve the coverage beyond what we already have: 92%. What does this test that the current test doesn't? Unless you can explain it, please remove these changes. TY.

Reply:
As the implementation has some checks and different behavior for 1 character or more than 1 character it tries to cover a few more branches, i.e. lines which are already covered, but some of the "if-conditions" not covered.